recipe/bld.bat

```diff
@@ -0,0 +1,28 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      -D NCO_MSVC_USE_MT=no ^
```

Comment:
Can you change this to `yes`? (Then there's no need for changing antlr)

Reply:
To be honest I don't know. I'll try this first and, if does not work, I'll change `antlr`.